oat-sa/extension-tao-itemqti
# feat/AUT-115/Multiple table cell selection

Related to: https://oat-sa.atlassian.net/browse/AUT-115

**Depends on:**
- [x] https://github.com/oat-sa/tao-core-shared-libs-fe/pull/16
- [x] https://github.com/oat-sa/tao-core/pull/3386
- [x] version of tao-core should be updated in composer.json

**How to test:**

- go to Item Authoring
- add A-block or interaction
- add table
- set focus outside
- try to set focus on table without activating container
- select more then one cell
- Insert and Links should be hidden in ckEditor toolbar

![image](https://user-images.githubusercontent.com/25976342/152320537-9f28c3a1-30e1-469e-90c7-be9289af9aa5.png)
